src/index.html

```diff
@@ -22,6 +22,298 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        class="logo"
+        src="./images/logo.png"
+        alt="Moyo Logo"
+      />
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__link">Samsung</li>
```

Comment:
Every word should be a link, fix it everywhere
```suggestion
          <li class="nav__item">
            <a
              class="nav__link is-active"
              href="#apple"
            >
              Apple
            </a>
          </li>
          <li class="nav__item">
            <a class="nav__link" href="#samsung">Samsung</a>
          </li>
```